localisation/00_memewc_united_localisation.csv

```diff
@@ -0,0 +1,96 @@
+#CODE;ENGLISH;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+amulet_1;[GetArtifactJewelryCommonPrefix] Necklace;;;;;;;;;;;;;x
+amulet_1_desc;Common Necklace.;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+amulet_2_diplomacy;[GetArtifactJewelryUncommonDiplomacyPrefix] Necklace[GetArtifactJewelryUncommonDiplomacySuffix];;;;;;;;;;;;;x
+amulet_2_diplomacy_desc;Uncommon Necklace.;;;;;;;;;;;;;x
+amulet_2_martial;[GetArtifactJewelryUncommonMartialPrefix] Necklace[GetArtifactJewelryUncommonMartialSuffix];;;;;;;;;;;;;x
+amulet_2_martial_desc;Uncommon Necklace.;;;;;;;;;;;;;x
+amulet_2_stewardship;[GetArtifactJewelryUncommonStewardshipPrefix] Necklace[GetArtifactJewelryUncommonStewardshipSuffix];;;;;;;;;;;;;x
+amulet_2_stewardship_desc;Uncommon Necklace.;;;;;;;;;;;;;x
+amulet_2_intrigue;[GetArtifactJewelryUncommonIntriguePrefix] Necklace[GetArtifactJewelryUncommonIntrigueSuffix];;;;;;;;;;;;;x
+amulet_2_intrigue_desc;Uncommon Necklace.;;;;;;;;;;;;;x
+amulet_2_learning;[GetArtifactJewelryUncommonLearningPrefix] Necklace[GetArtifactJewelryUncommonLearningSuffix];;;;;;;;;;;;;x
+amulet_2_learning_desc;Uncommon Necklace.;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+amulet_3_diplomacy;[GetArtifactJewelryRareDiplomacyPrefix] Necklace[GetArtifactJewelryRareDiplomacySuffix];;;;;;;;;;;;;x
+amulet_3_diplomacy_desc;Rare Necklace.;;;;;;;;;;;;;x
+amulet_3_martial;[GetArtifactJewelryRareMartialPrefix] Necklace[GetArtifactJewelryRareMartialSuffix];;;;;;;;;;;;;x
+amulet_3_martial_desc;Rare Necklace.;;;;;;;;;;;;;x
+amulet_3_stewardship;[GetArtifactJewelryRareStewardshipPrefix] Necklace[GetArtifactJewelryRareStewardshipSuffix];;;;;;;;;;;;;x
+amulet_3_stewardship_desc;Rare Necklace.;;;;;;;;;;;;;x
+amulet_3_intrigue;[GetArtifactJewelryRareIntriguePrefix] Necklace[GetArtifactJewelryRareIntrigueSuffix];;;;;;;;;;;;;x
+amulet_3_intrigue_desc;Rare Necklace.;;;;;;;;;;;;;x
+amulet_3_learning;[GetArtifactJewelryRareLearningPrefix] Necklace[GetArtifactJewelryRareLearningSuffix];;;;;;;;;;;;;x
+amulet_3_learning_desc;Rare Necklace.;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+amulet_4_diplomacy;[GetArtifactJewelryEpicDiplomacyPrefix] Necklace[GetArtifactJewelryEpicDiplomacySuffix];;;;;;;;;;;;;x
+amulet_4_diplomacy_desc;Mythical Necklace.;;;;;;;;;;;;;x
+amulet_4_martial;[GetArtifactJewelryEpicMartialPrefix] Necklace[GetArtifactJewelryEpicMartialSuffix];;;;;;;;;;;;;x
+amulet_4_martial_desc;Mythical Necklace.;;;;;;;;;;;;;x
+amulet_4_stewardship;[GetArtifactJewelryEpicStewardshipPrefix] Necklace[GetArtifactJewelryEpicStewardshipSuffix];;;;;;;;;;;;;x
+amulet_4_stewardship_desc;Mythical Necklace.;;;;;;;;;;;;;x
+amulet_4_intrigue;[GetArtifactJewelryEpicIntriguePrefix] Necklace[GetArtifactJewelryEpicIntrigueSuffix];;;;;;;;;;;;;x
+amulet_4_intrigue_desc;Mythical Necklace.;;;;;;;;;;;;;x
+amulet_4_learning;[GetArtifactJewelryEpicLearningPrefix] Necklace[GetArtifactJewelryEpicLearningSuffix];;;;;;;;;;;;;x
+amulet_4_learning_desc;Mythical Necklace.;;;;;;;;;;;;;x
```

Comment:
Why mythical instead of epic?

Reply:
I thought it will be sound better that way, but I can revert it back to epic.

Reply:
Changed